arrow-fx/src/main/kotlin/arrow/fx/Promise.kt

```diff
@@ -103,7 +103,7 @@ interface Promise<F, A> {
    *       p.complete(2)
    *     }
    *   }.attempt().unsafeRunSync() ==
-   *     IO.raiseError<Int>(Promise.AlreadyFulfilled).attempt().unsafeRunSync()
+   *     IO.raiseException<Int>(Promise.AlreadyFulfilled).attempt().unsafeRunSync()
```

Comment:
This...doesn't seem right. The generic should be of the type of `Promise.AlreadyFulfilled` instead, right?

Reply:
Oh nevermind, it's an attempt. Ye.